universal-login-commons/lib/core/models/authorisation.ts

```diff
@@ -0,0 +1,7 @@
+export interface CancelAuthorisationRequest {
+  walletContractAddress: string;
+  key: string;
```

Comment:
Shall we add signature field here?
```ts
export interface CancelAuthorisationRequest {
  walletContractAddress: string;
  key: string;
  signature: string?;
}
```

Reply:
rename `key` -> `publicKey`

Reply:
Agree. Also I've noticed that address and publicKey are used interchangeably 
[ekhm...](https://github.com/UniversalLogin/UniversalLoginSDK/blob/1fc6e900c05cc73eaa416ddde68cad75b9007855/universal-login-wallet/src/core/services/ConnectToWallet.ts#L8)

Reply:
By convention, we use `publicKey`, when it does not store funds.